src/components/tooltip/bl-tooltip.test.ts

```diff
@@ -25,13 +26,13 @@ describe('bl-tooltip', () => {
        class="trigger"
        name="tooltip-trigger">
       </slot>
-      <div class='tooltip'>
+      <div class='wrapper'>
```

Comment:
Why do we need this wrapper?

Reply:
I have used wrapper class to set display as inline-block. If I don't use it every tooltip displayed on single row(block). But maybe am I missing something?